apps/ua-parsing/src/main.js

```diff
@@ -0,0 +1,124 @@
+import { UAParser } from './ua-parser.js';
```

Comment:
why not use `https://www.npmjs.com/package/ua-parser-js`?

Reply:
We’re actually using 99% of the source code of that package, [but its current implementation has very outaded special exports to handle cases where it’s invoked from jQuery](https://github.com/faisalman/ua-parser-js/blob/master/src/ua-parser.js#L1132-L1153), so it was easier to lift the internal part to generate code that could run inside BigQuery.